step.yml

```diff
@@ -289,8 +300,8 @@ inputs:
       category: Debug
       title: "Should retry test on failure?"
       description: |-
-       If you set this input to `yes`, the Step will rerun your tests. With Xcode 13 and above, only your failed test cases will be rerun. With older Xcode versions, all test cases will be rerun. 
-        Please note that this feature is only available from Xcode 13 and above, earlier versions support rerunning ALL test cases instead of a specific failed one.  
+       If you set this input to `yes`, the Step will rerun your tests. With Xcode 13 and above, only your failed test cases will be rerun. With older Xcode versions, all test cases will be rerun.
+        Please note that this feature is only available from Xcode 13 and above, earlier versions support rerunning ALL test cases instead of a specific failed one.
```

Comment:
correct me if I'm wrong but I think the previous two sentences say exactly the same so I dont think we need these two sentences: "Please note that this feature is only available from Xcode 13 and above, earlier versions support rerunning ALL test cases instead of a specific failed one."

Reply:
Exactly, I just noticed this as well! Let's fix this in the next PR, as this step is going to change anyway.

Reply:
Thank you :)